### PR TITLE
use cross platform path separatator in build

### PR DESCRIPTION
**Pull request by martijnhoekstra**
Hello, world!

build instructions now work on Windows.

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -453,7 +453,7 @@ object Build {
   def findLib(attList: Seq[Attributed[File]], name: String) = attList
     .map(_.data.getAbsolutePath)
     .find(_.contains(name))
-    .toList.mkString(":")
+    .toList.mkString(java.io.File.pathSeparatorChar.toString)
```
Comment (sjrd) [inline project/Build.scala]: `pathSeparator` can be used instead of `pathSeparatorChar.toString`.

File: project/Build.scala [hunk 2]
```diff
@@ -453,7 +453,7 @@ object Build {
   def findLib(attList: Seq[Attributed[File]], name: String) = attList
     .map(_.data.getAbsolutePath)
     .find(_.contains(name))
-    .toList.mkString(":")
+    .toList.mkString(java.io.File.pathSeparator)
```
Comment (smarter) [inline project/Build.scala]: I would `import java.io.File`.

**Comment by martijnhoekstra** [comment]
I think I caught all platform-incompatible uses of `':'` and `":"`  as path separators. There may be more lurking somewhere as parts of hardcoded strings.

**Comment by smarter** [comment]
Thanks!

**Pull request merged**